### PR TITLE
suricata: Configure the Intrusion Detection System

### PR DESCRIPTION
**Pull request by lmacken**
This configures suricata and pulls down the latest GPL'd ETOpen Ruleset.

It also performs some network optimizations, and I was finally able to get suricata to run without any warnings for the first time. It also has the ability to disable certain rule goups.

### CODE REVIEW

File: roles/security/tasks/suricata.yml [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+  shell: nmcli -t -f device device | head -n1
+  register: iface
+  changed_when: False
+  tags:
+  - security
+  - ids
+
+- name: suricata | Downloading the latest ETOpen Ruleset http://emergingthreats.net/open-source/etopen-ruleset
+  get_url:
+    url=http://rules.emergingthreats.net/open/suricata/emerging.rules.tar.gz
```
Comment (ralphbean) [inline roles/security/tasks/suricata.yml]: Is this available over https?

File: roles/security/tasks/suricata.yml [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+    dest=/tmp/emerging.rules.tar.gz
```
Comment (ralphbean) [inline roles/security/tasks/suricata.yml]: And, can we `/var/tmp` instead of `/tmp`?

File: roles/security/tasks/suricata.yml [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+    src=/tmp/emerging.rules.tar.gz
+    dest=/etc/suricata
+    force=yes
+  tags:
+  - security
+  - ids
+
+- name: suricata | Downloading the latest ETOpen configs
+  get_url:
+    url="http://rules.emergingthreats.net/open/suricata/{{ item }}"
```
Comment (ralphbean) [inline roles/security/tasks/suricata.yml]: Same question here about https.

**Comment by ralphbean** [comment]
This will be :money_with_wings:.

**Comment by lmacken** [comment]
changed to /var/tmp and https.

**Pull request merged**